docs/fans.md

```diff
@@ -0,0 +1,36 @@
+﻿## ** What is Shenanigan? ** ![woman_cartwheeling](https://lh6.googleusercontent.com/lJUzNtQP47lexQBdxruB084UG0G2IzRM_rN1tdIaUXNGQF4PMhT0che_hmY8GbxH6q2_Qf9bl4pzT4RXfaLH7iNquGHhv1VfcIWTKEJnCU_rbURBRts0_EGaG56GNaW5yIg9OPEG)
```

Comment:
When you use emojis, make sure you use the ascii variant

to do this you can press `CMD +CTRL + Space (Win + . for windows)`

Reply:
I'm on a Chrome book doing this, so I will find out how to do this